internal/statesync/snapshotter.go

```diff
@@ -43,13 +43,15 @@ const (
 type Snapshotter struct {
 	dbConfig    *DBConfig
 	snapshotDir string
+	MaxRowSize  int
```

Comment:
Really no need to export the field, unless we want to set it from the caller directly rather than via the constructor, which would be fine too.

Reply:
ah, autocorrected. changing back